package.json

```diff
@@ -6,19 +6,22 @@
     "Rackspace Web Team <rswebdev@rackspace.com>"
   ],
   "scripts": {
+    "babel": "rm -rf styleguide/js && babel styleguide/global styleguide/derek --out-dir styleguide/js",
+    "gh-pages": "./scripts/build-pages.sh",
     "harp": "harp ",
     "harp:compile": "npm run harp compile styleguide dist",
-    "start": "concurrently 'harp server styleguide' 'browser-sync start --config bs-config.js'",
-    "gh-pages": "./scripts/build-pages.sh",
+    "hooks": "./scripts/hooks.sh",
+    "lint:js": "eslint styleguide/derek styleguide/global",
+    "lint:sass": "sass-lint -v",
     "preversion": "npm run babel && npm run harp compile styleguide dist && git add -f dist",
+    "postinstall": "npm run hooks && npm run babel",
     "postversion": "git push && git push --tags",
     "sass:compile": "node-sass --output-style=compressed --quiet -o /tmp styleguide/css",
-    "sass-lint": "sass-lint -v",
-    "hooks": "./scripts/hooks.sh",
-    "postinstall": "npm run hooks",
-    "js-lint": "eslint styleguide/derek styleguide/global",
-    "babel": "rm -rf styleguide/js && babel styleguide/global styleguide/derek --out-dir styleguide/js",
-    "babel:watch": "babel styleguide/global styleguide/derek --out-dir styleguide/js --watch"
+    "start": "concurrently 'npm run start:harp' 'npm run start:bs' 'npm run watch:babel' 'npm run watch:sass'",
+    "start:harp": "harp server styleguide",
```

Comment:
😎  nice. I see what you did here. Imho I think having the commands to start harp and browser sync documented here in case there is a need to start them separately. . . If it doesn't hurt anything to have them here. awesome. also. 
Also dig the watch:babel - watch:sass  change, same as lint..